### DISCUSSION TITLE
PrimeFaces 8.0 Showcase Down

### DISCUSSION THREAD

**Discussion by F1shZ** [primefaces/community#2031] [PrimeFaces] [answered]
The pages are all returning errors, could you please check? Thank you!

https://www.primefaces.org/showcase-v8/error.xhtml

**Comment by melloware** [comment, marked as answer]
PrimeTek has decided to remove that showcase as they are only supporting LTS deployments of the showcase.

You can download and run the showcase locally with this zip file: https://github.com/user-attachments/files/15586639/primefaces-showcase-8.0.zip

just run `mvn clean jetty:run`

**Reply by melloware** [reply]
Here is the download so you can run it locally:  https://github.com/primefaces/primefaces-showcase-legacy/files/9189953/primefaces-showcase-8.0.zip

**Reply by MocniMax** [reply]
Hi.

I am trying to build and run Primefaces 8 showcase from mentioned zip but am getting following error:

```
[INFO] Artifact org.primefaces.themes:all-themes:pom:1.0.10 is present in the local repository, but cached from a remote repository ID that is unavailable in current build context, verifying th
at is downloadable from [maven-default-http-blocker (http://0.0.0.0/, default, releases+snapshots, blocked), central (https://repo.maven.apache.org/maven2, default, releases)]
[INFO] Artifact org.primefaces.themes:all-themes:pom:1.0.10 is present in the local repository, but cached from a remote repository ID that is unavailable in current build context, verifying th
at is downloadable from [maven-default-http-blocker (http://0.0.0.0/, default, releases+snapshots, blocked), central (https://repo.maven.apache.org/maven2, default, releases)]
[INFO] Artifact net.sf.barcode4j:barcode4j-light:pom:2.1 is present in the local repository, but cached from a remote repository ID that is unavailable in current build context, verifying that
is downloadable from [maven-default-http-blocker (http://0.0.0.0/, default, releases+snapshots, blocked), central (https://repo.maven.apache.org/maven2, default, releases)]
[INFO] Artifact net.sf.barcode4j:barcode4j-light:pom:2.1 is present in the local repository, but cached from a remote repository ID that is unavailable in current build context, verifying that
is downloadable from [maven-default-http-blocker (http://0.0.0.0/, default, releases+snapshots, blocked), central (https://repo.maven.apache.org/maven2, default, releases)]
[INFO] ------------------------------------------------------------------------
[INFO] BUILD FAILURE
[INFO] ------------------------------------------------------------------------
[INFO] Total time:  1.565 s
[INFO] Finished at: 2024-06-05T13:18:09+02:00
[INFO] ------------------------------------------------------------------------
[ERROR] Failed to execute goal on project showcase: Could not resolve dependencies for project org.primefaces:showcase:war:8.0-SNAPSHOT: Failed to collect dependencies at org.primefaces.themes:
all-themes:jar:1.0.10: Failed to read artifact descriptor for org.primefaces.themes:all-themes:jar:1.0.10: The following artifacts could not be resolved: org.primefaces.themes:all-themes:pom:1.
0.10 (present, but unavailable): Could not transfer artifact org.primefaces.themes:all-themes:pom:1.0.10 from/to maven-default-http-blocker (http://0.0.0.0/): Blocked mirror for repositories: [
prime-repo (http://repository.primefaces.org, default, releases+snapshots)] -> [Help 1]

```
Neither of solutions I found solve my issue.
Any idea where the problem is?

Thanks.

**Reply by melloware** [reply]
Yes Maven has gotten stricter over the years and only wants HTTPS: so change this in the pom.xml to HTTPS:

```xml
    <repositories>
        <repository>
            <id>prime-repo</id>
            <name>PrimeFaces Maven Repository</name>
            <url>https://repository.primefaces.org</url>
        </repository>
    </repositories>
```

**Reply by MocniMax** [reply]
Yes, I just realized it, thanks a lot.

Now it builds correctly.

Thanks a lot.

**Reply by MocniMax** [reply]
Now I have another problem, proper layout is missing.
Here is how it looks like when deployed on Jetty 

![image](https://github.com/primefaces/community/assets/26117076/e9350006-6a65-4cdb-9cb9-8a76a92c8d85)
and menu

![image](https://github.com/primefaces/community/assets/26117076/c23ca2eb-8ab1-4bd0-87c6-0eebb13b2b24)


Actual component showcase can be used but not in the correct way

![image](https://github.com/primefaces/community/assets/26117076/fca7f4b8-94e7-4d65-ba77-c04432b16824)

Any idea how to solve it?

**Reply by melloware** [reply]
let me look.

**Reply by melloware** [reply]
Try this version:
[primefaces-showcase-8.0.zip](https://github.com/user-attachments/files/15586639/primefaces-showcase-8.0.zip)

Run `mvn clean jetty:run` to run locally or `mvn clean package` to build the WAR.  I just deployed the war on Jetty and its working fine.

**Reply by MocniMax** [reply]
Building it ...

**Reply by MocniMax** [reply]
Yes, it runs correctly now!

![image](https://github.com/primefaces/community/assets/26117076/2dc1d490-bbc5-4e9a-b6f4-4912964f4c1e)

Thanks a lot!!

**Reply by melloware** [reply]
Nice!

**Reply by MocniMax** [reply]
I successfully built showcase 10 with the same change from http to https.

**Reply by jepsar** [reply]
@melloware "they are only supporting LTS deployments of the showcase". That's not true (anymore?). They only deploy a recent version of the latest community release.

**Reply by melloware** [reply]
@jepsar it looks like you are correct latest version only.

**Comment by miroskow** [comment]
To complete the set, I'm still looking, primefaces-extensions-8.0.5 showcase.
@melloware help :)
Thanks a lot

**Reply by melloware** [reply]
Here is the PFE showcase for 8.0. 
[pfe-showcase-8.0.zip](https://github.com/user-attachments/files/16052013/pfe-showcase-8.0.zip)

**Comment by diasthi31** [comment]
![Captura de Tela 2024-07-15 às 15 29 46](https://github.com/user-attachments/assets/87cb16d3-d803-42ce-a597-14c4508107c9)
 
I followed the steps mentioned above, but any tab I try to enter gives an error.

**Reply by melloware** [reply]
do you see an error in the console?   I just tried it and `mvn clean jetty:run` is working fine for me?

**Reply by diasthi31** [reply]
![Captura de tela de 2024-07-15 15-57-37](https://github.com/user-attachments/assets/6e2c9b0c-b793-4f60-a7a2-1ab1370afc1b)

Whenever clicking on a menu item generates an error similar to this.

**Reply by melloware** [reply]
Add this to the pom.xml and try again...

```xml
		<dependency>
			<groupId>com.sun.xml.bind</groupId>
			<artifactId>jaxb-impl</artifactId>
			<version>2.3.6</version>
		</dependency>
```

**Reply by diasthi31** [reply]
It worked perfectly, thank you very much.

**Comment by erickdeoliveiraleal** [comment]
@melloware please can you add the 10.0 showcase? 
I asked for it here: https://github.com/orgs/primefaces/discussions/4336

**Reply by melloware** [reply]
10.0.x: https://github.com/user-attachments/files/22523631/primefaces-showcase-10.0.x.zip

**Reply by melloware** [reply]
The showcases have all been retutned to GitHub Tags:

https://github.com/primefaces/primefaces/tree/8.X
